addon/components/o-s-s/star-rating.ts

```diff
@@ -1,11 +1,14 @@
 import Component from '@glimmer/component';
 import { assert } from '@ember/debug';
+import { action } from '@ember/object';
 
-interface OSSStarRatingArgs {
+export interface OSSStarRatingArgs {
```

Comment:
Why export it?

Reply:
For consistency between onChange event in the OSSStarRatingArgs interface &  the one in starRating. 
The interface in Attribute::Rating now has `onChange?: OSSStarRatingArgs['onChange'];` .

Reply:
Since we don't use onChange on Attr::Rating, I removed the export @JulienVannier66